README.md

```diff
@@ -13,6 +13,14 @@ Includes functions for posterior analysis, model checking, comparison and diagno
 The official ArviZ documentation can be found here
 https://arviz-devs.github.io/arviz/index.html
```

Comment:
Not your change, not entirely necessary, but can we change this to an inline link, and combine these three short sentences into one paragraph (they'll need periods at the end of each)? Here's the link I mean:

The official ArviZ documentation can be found [here](https://arviz-devs.github.io/arviz/index.html).